internal/component/common/loki/positions/positions_test.go

```diff
@@ -37,6 +39,90 @@ func tempFilename(t *testing.T) string {
 	return name
 }
 
+func TestConversion(t *testing.T) {
+	tmpDir := t.TempDir()
```

Comment:
nit: There's a good bit of similar code that maybe could be pulled into a common util func.